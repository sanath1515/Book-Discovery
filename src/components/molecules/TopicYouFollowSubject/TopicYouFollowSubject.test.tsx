
import { render } from "@testing-library/react";
import TopicYouFollowSub from "./TopicYouFollowSubject";

describe("Topics renders correctly", () => {
  test("should match shapshot", () => {
    const { asFragment } = render(
      <TopicYouFollowSub imgSrc="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/50247E5B-3788-492D-956F-63172A8FDBF8.png" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
