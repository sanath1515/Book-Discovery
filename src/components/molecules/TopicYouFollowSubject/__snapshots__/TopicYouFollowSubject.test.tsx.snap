// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Topics renders correctly should match shapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root"
  >
    <div
      class="MuiGrid-root"
    >
      <img
        class="makeStyles-Rectangle-1"
        src="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/50247E5B-3788-492D-956F-63172A8FDBF8.png"
      />
    </div>
    <div
      class="MuiGrid-root"
    >
      <h6
        class="MuiTypography-root makeStyles-Subject-2 MuiTypography-h6"
      />
    </div>
  </div>
</DocumentFragment>
`;
